container/nextcloud/Dockerfile

```diff
@@ -0,0 +1,85 @@
+FROM archlinux:latest
+MAINTAINER Sebastian Selka
+
+RUN pacman --noconfirm -Syu
+RUN pacman --noconfirm -S \
+        supervisor \
+        nginx-mainline \
+        wget \
+        php-legacy-fpm \
+        php-legacy-pgsql \
+        php-legacy-apcu \
+        php-legacy-redis \
+        php-legacy-gd \
+        nextcloud \
+        fcron
+
+EXPOSE 80 80
+
+RUN cd /usr/share/webapps/nextcloud/apps/ && \
```

Comment:
Ich würde sämtliche Versionen als z.B. `ARG CALENDAR_VERSION=v4.7.11` oben in den Container packen. Dann kannst das interpolieren und hast nur eine Stelle, wo man das ändern muss

Reply:
Done